apps/estuary/config/test.exs

```diff
@@ -1,6 +1,17 @@
 use Mix.Config
 
+config :logger, level: :warn
+
 config :estuary,
-  elsa_endpoint: nil
+  topic_reader: MockReader,
+  table_writer: MockTable,
+  endpoints: [localhost: 9092],
```

Comment:
is this still needed for the mocks?  Just curious.

Reply:
Dunno. We'll check and remove it if our tests still pass.

Reply:
Removed endpoints.